source/scss/_pf-variables.scss

```diff
@@ -326,243 +411,291 @@ $table-sm-cell-padding:         $pf-spacer-xxxs !default;
+// $custom-file-padding-x:     .5rem !default;
+// $custom-file-padding-y:     1rem !default;
+// $custom-file-line-height:   1.5 !default;
+// $custom-file-color:         #555 !default;
+// $custom-file-bg:            #fff !default;
+// $custom-file-border-width:  $border-width !default;
+// $custom-file-border-color:  #ddd !default;
+// // $custom-file-border-radius: $border-radius !default;
+// $custom-file-box-shadow:    inset 0 .2rem .4rem rgba(0,0,0,.05) !default;
+// $custom-file-button-color:  $custom-file-color !default;
+// $custom-file-button-bg:     #eee !default;
+// $custom-file-text: (
+//   placeholder: (
+//     en: "Choose file..."
+//   ),
+//   button-label: (
+//     en: "Browse"
+//   )
+// ) !default;
 
-$form-icon-danger-color: $brand-danger !default;
-$form-icon-danger: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='#{$form-icon-danger-color}' viewBox='-2 -2 7 7'%3E%3Cpath stroke='%23d9534f' d='M0 0l3 3m0-3L0 3'/%3E%3Ccircle r='.5'/%3E%3Ccircle cx='3' r='.5'/%3E%3Ccircle cy='3' r='.5'/%3E%3Ccircle cx='3' cy='3' r='.5'/%3E%3C/svg%3E") !default;
 
+// // Form validation icons
+// $form-icon-success-color: $brand-success !default;
+// $form-icon-success: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3E%3Cpath fill='#{$form-icon-success-color}' d='M2.3 6.73L.6 4.53c-.4-1.04.46-1.4 1.1-.8l1.1 1.4 3.4-3.8c.6-.63 1.6-.27 1.2.7l-4 4.6c-.43.5-.8.4-1.1.1z'/%3E%3C/svg%3E") !default;
 
+// $form-icon-warning-color: $brand-warning !default;
+// $form-icon-warning: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3E%3Cpath fill='#{$form-icon-warning-color}' d='M4.4 5.324h-.8v-2.46h.8zm0 1.42h-.8V5.89h.8zM3.76.63L.04 7.075c-.115.2.016.425.26.426h7.397c.242 0 .372-.226.258-.426C6.726 4.924 5.47 2.79 4.253.63c-.113-.174-.39-.174-.494 0z'/%3E%3C/svg%3E") !default;
 
-// Buttons
+// $form-icon-danger-color: $brand-danger !default;
+// $form-icon-danger: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='#{$form-icon-danger-color}' viewBox='-2 -2 7 7'%3E%3Cpath stroke='%23d9534f' d='M0 0l3 3m0-3L0 3'/%3E%3Ccircle r='.5'/%3E%3Ccircle cx='3' r='.5'/%3E%3Ccircle cy='3' r='.5'/%3E%3Ccircle cx='3' cy='3' r='.5'/%3E%3C/svg%3E") !default;
+
+
+
+// Dropdowns
 //
-// For each of Bootstraps buttons, define text, background and border color.
+// Dropdown menu container and contents.
 
-$btn-padding-x:                  $pf-spacer-md !default;
-$btn-padding-y:                  $pf-spacer-xs !default;
-$btn-line-height:                1.25 !default;
-$btn-font-weight:                $pf-font-weight-regular !default;
-$btn-box-shadow:                 inset 0 1px 0 rgba($pf-color-white, .15), $pf-box-shadow-sm;
+$dropdown-min-width:             $pf-spacer-xxxxl !default;
```

Comment:
I did not expect to see a -spacer as a width. Can we reserve spacers for margins and padding and specify this another way?